styleguide/components/11-base/boxes/_boxes.scss

```diff
@@ -84,6 +86,8 @@
       width: 6rem;
       height: 6rem;
       margin: auto;
+      border-radius: 50%;
+      box-shadow: 0 10px 35px -25px;
```

Comment:
Zouden we dit niet afsplitsen zodat het herbruikbaar is voor andere spot images?

Reply:
Wordt enkel gebruikt op die van box-top en box-left op mobiel.